include/ginkgo/core/base/exception_helpers.hpp

```diff
@@ -89,17 +89,21 @@ namespace gko {
 
 
 /**
- * Creates a NotSupported exception.
+ * Throws Creates a NotSupported exception.
```

Comment:
```suggestion
 * Throws a NotSupported exception.
```

Reply:
Thanks for noticing, guess I edited it without proofreading...
Fixed it in the latest commit.